front_end/panels/timeline/TimelineUIUtils.ts

```diff
@@ -1708,6 +1729,27 @@ export class TimelineUIUtils {
     return contentHelper.fragment;
   }
 
+  /**
+   * Get attribution data for a given URL.
+   *
+   * @param {string} url URL to check for attribution data.
+   * @param {Trace.Types.Events.PerformanceAttribution[]} attributions Array of performance attributions. Immutable.
+   *
+   * @return {string|null} Attribution name or null if no attribution is found.
+   */
+  static getAttributionForUrl(url: string, attributions: Trace.Types.Events.PerformanceAttribution[]): string|null {
+    const attribution = attributions.find(attribution => {
+      const detail = JSON.parse(attribution.args.data.detail);
+      const parsedUrl = new URL(url);
+      return detail.path.includes(parsedUrl.pathname);
+    });
+    if (attribution) {
+      const detail = JSON.parse(attribution.args.data.detail);
+      return 'core' === detail.slug ? i18nString(UIStrings.wordpressCore) : i18nString(UIStrings.wordpressPlugin) + ' - ' + detail.name;
```

Comment:
@todo add support for "Active Theme" as well

Reply:
Added theme enqueue data on the WordPress side in [WordPress/performance@`3e5b038` (#1629)](https://github.com/WordPress/performance/pull/1629/commits/3e5b038499b3d40df0d616577b41607983b51c09)

Reply:
Added theme identification in [`7a5caa6` (#192)](https://github.com/ChromeDevTools/devtools-frontend/pull/192/commits/7a5caa6f87b923993d75196bd95a6cc779feab2f)